modules/33-services

```diff
@@ -4,19 +4,20 @@ set -euo pipefail
 
 # Default services, have to be declared before sourcing framework.sh so the user can override them
 declare -A services
-services["nginx"]="nginx"
-services["Docker"]="docker"
-services["SSH"]="sshd"
-services["Fail2Ban"]="fail2ban"
-services["UFW"]="ufw"
+services["nginx"]="Nginx"
+services["docker"]="Docker"
+services["sshd"]="SSH"
+services["fail2ban"]="Fail2Ban"
+services["ufw"]="UFW"
 
 source "$BASE_DIR/framework.sh"
 
 statuses=()
 for key in "${!services[@]}"; do
-    # systemctl is-active returns non-zero code if service is inactive
-    set +e; status=$(systemctl is-active ${services[$key]}); set -e
-    statuses+=("$(print_status "$key" "$status")")
+    if [ $(systemctl list-unit-files "$key.service" | wc -l) -gt 3 ]; then
+        status=$(systemctl is-active $key.service)
```

Comment:
Hi, please apply this change:
```suggestion
        # systemctl is-active returns non-zero code if service is inactive
        set +e; status=$(systemctl is-active $key.service); set -e
```
This is necessary because of `set -euo pipefail` in the beginning of the file - I want modules to exit immediately on any error. However, `systemctl is-active` returns non-zero code when a service is inactive. Because of this, without `set +e` the module won't be displayed if any of the services is inactive.

Reply:
I assumed the error only happens if it does not exist. I don't get that behavior on my maschines.

Have you tried `systemctl show -p ActiveState --value NAME.service` this might not throw an exit code?

Reply:
This is interesting. For me, with `systemd` v248, this is what `systemctl(1)` says:

> is-active PATTERN...
>            Check whether any of the specified units are active (i.e. running). Returns an exit code 0 if at least one is active, or non-zero otherwise. Unless --quiet is specified, this will also print the current unit state to standard output.

Either way, your solution with `systemctl show` works properly and is more elegant anyway so please go ahead and change this if you don't mind.

Reply:
So I had a moment and did it myself, thanks for your help!